oxidation/libparsec/crates/serialization_format/src/lib.rs

```diff
@@ -53,6 +54,13 @@ pub fn parsec_protocol_cmds_family(path: TokenStream) -> TokenStream {
 
     let mut json_cmds = vec![];
     for json_path in dir.filter_map(|entry| entry.ok()) {
+        // Only keep .json/.json5 files
+        let file_name = json_path.file_name();
+        match std::path::Path::new(&file_name).extension() {
+            Some(x) if x == OsStr::new("json5") || x == OsStr::new("json") => (),
```

Comment:
FYI, you don't need to call `OsStr::new`, because `OsStr` implements `PartialEq` for `str`

Reply:
:+1: done